packages/destination-actions/src/destinations/stackadapt-audiences/forwardProfile/functions.ts

```diff
@@ -0,0 +1,35 @@
+import { RequestClient } from '@segment/actions-core'
+import { Payload } from './generated-types'
+
+export async function performForwardProfiles(request: RequestClient, events: Payload[]) {
+  // TODO: switch to production endpoint
+  const endpoint = 'https://sandbox.stackadapt.com/public/graphql'
```

Comment:
we can use this variable at the top level https://github.com/StackAdapt/action-destinations/blob/2b3bd37c1e750f0bf7aa25713897891ea0e0e243/packages/destination-actions/src/destinations/stackadapt-audiences/index.ts#L7